core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1209,7 +1216,11 @@ class GroupCoordinator(val brokerId: Int,
   }
 
   private def removeHeartbeatForLeavingMember(group: GroupMetadata, member: MemberMetadata): Unit = {
-    val memberKey = MemberKey(group.groupId, member.memberId)
```

Comment:
I don't think we should keep this overloading method here. We can just remove it and change all the caller to pass memberId (I searched, only 2 places called it)

Reply:
Make sense.